ttnn/cpp/ttnn/operations/conv/conv2d/device/multi_core_optimized_conv_sharded/optimized_conv_op_sharded_v2.cpp

```diff
@@ -1203,6 +1215,11 @@ operation::ProgramWithCallbacks multi_core_optimized_conv_sharded_v2_impl(
         compute_defines["PACKER_L1_ACC"] = "1";
     }
 
+    if(use_non_tile_height){
```

Comment:
can we just use constexpr in kernels instead of passing in new defines/

Reply:
Sure.